Core/include/Acts/Seeding/BinnedSPGroup.ipp

```diff
@@ -156,45 +155,40 @@ Acts::BinnedSPGroup<external_spacepoint_t>::BinnedSPGroup(
   // keep track of changed bins while sorting
   boost::container::flat_set<size_t> rBinsIndex;
 
-  std::size_t counter = 0;
-  for (spacepoint_iterator_t it = spBegin; it != spEnd; it++, ++counter) {
-    if (*it == nullptr) {
-      continue;
-    }
-    const external_spacepoint_t& sp = **it;
-    const auto& [spPosition, variance] =
-        toGlobal(sp, config.zAlign, config.rAlign, config.sigmaError);
+  for (spacepoint_iterator_t it = spBegin; it != spEnd; ++it) {
+    const external_spacepoint_t& sp = *it;
 
-    float spX = spPosition[0];
-    float spY = spPosition[1];
-    float spZ = spPosition[2];
+    float spX = sp.x();
+    float spY = sp.y();
+    float spZ = sp.z();
+    Acts::Vector3 spPosition{spX, spY, spZ};
 
     // store x,y,z values in extent
-    rRangeSPExtent.extend({spX, spY, spZ});
+    rRangeSPExtent.extend(spPosition);
 
+    // why do it here after the extend????
```

Comment:
I agree that it makes more sense to fill the extent after these cuts, I tested this once for ITk and it was a minor change

Reply:
but maybe something for another PR

Reply:
sure, do you wanna address it?

Reply:
yes I can do it